cmd/kops/gen_help_docs.go

```diff
@@ -21,6 +21,11 @@ import (
 	"github.com/spf13/cobra/doc"
 )
 
+const fileHeader = `
+<!--- This file is automatically generated by make gen-cli-docs; changes should be made in the go CLI command code -->
```

Comment:
> go CLI command code
Should we say where the code is?  cmd/pkg/?

There are a lot of first timers that are making these changes.  Make it simple for them.